NGitLab.Mock/Config/GitLabCommit.cs

```diff
@@ -30,6 +30,11 @@ public class GitLabCommit : GitLabObject<GitLabProject>
         /// Target branch for a merge commit (required for merge commit)
         /// </summary>
         public string TargetBranch { get; set; }
+
+        /// <summary>
+        /// Indicates if source branch must be deleted after merge
+        /// </summary>
+        public bool DeleteSourceBranch { get; set; }
```

Comment:
- Should this property be in GitLabMergeRequest?
- I think it should be nullable and also exposed on the Project

    ![image](https://user-images.githubusercontent.com/509220/131847869-834773d9-2017-4e63-b886-daf900fd66b8.png)

Reply:
This property is only for Git graph and no related to merge requests. You can construct the project Git graph with config:

```csharp
var config = new GitLabConfig()
    .WithProject("MyProject", configure: project => project
        .WithCommit("Commit on main #1")
        .WithCommit("Commit on branch #2", branch: "branch")
        .WithCommit("Commit on main #3")
        .WithMergeCommit("branch", deleteSourceBranch: true));
```